src/core/server/elasticsearch/cluster_client.ts

```diff
@@ -64,7 +64,9 @@ const callAPI = async (
       const request = api.call(apiContext, clientParams);
       if (options.signal) {
         options.signal.addEventListener('abort', () => {
-          request.abort();
+          if (typeof request.abort === 'function') {
```

Comment:
workaround for https://github.com/elastic/kibana/issues/59712

Reply:
This shouldn't be required and may be hiding something bigger (https://github.com/elastic/kibana/issues/59712#issuecomment-598807096) Not sure if we should use the workaround for now or wait until the root cause has been identified @restrry ?

Reply:
I'd wait for an answer on the referenced comment. I'm scared that it could break some other less obvious things: a connection pool in elasticsearch client, for example.

Reply:
Agree, I want to hold off on merge with this until the underlying issue is resolved :)

Reply:
FYI, I've had a todo open for a long time to fix the root cause, but I've had too much on my plate lately. Once the following two PR's are merged we can release a new version of the Elastic APM Node.js Agent and we should be all good:

- https://github.com/elastic/apm-agent-nodejs/pull/1566
- https://github.com/elastic/apm-agent-nodejs/pull/1583

Reply:
I'm reverting this change so the PR can be merged, because it has some TS cleanups now that I want to get in. 

Having this issue left open in APM makes the overall feature of this PR a bit impractical, but when a fix is delivered to the APM Node agent, Reporting will automatically become measurable with APM.